embedding-sdk/unauthenticated/readme.md

```diff
@@ -0,0 +1,84 @@
+
+#### The features included in this demo are as follows:
+
+- Set the charts theme to either `'light'` or `'dark'`
+- Get the charts current theme
+- Set the charts refresh interval
+  - Note, the default interval is 0, implying that after rendering, the chart will never refresh.
+  - The minimum refresh interval is 10 seconds.
+- Get the charts current refresh interval
+- Manually refresh the chart
+- Set the charts current filter
+  - Note, filtering on a chart requires setting up white listed fields in MongoDB Charts. We have done this for our sample data.
+- Get the current filter on a chart
+
+## Preparing your Chart for Embedding
+
+---
+
+If you would like you use our sample data, feel free to skip to running the demo.
+
+Othwerise, do the following:
+
+1. Log onto MongoDB Charts
+
+2. If you haven't done so already, create a chart on any dashboard that you would like to embed.
+
+3. Go to the Data Sources tab, find the data source that you are using on the chart, and choose Embedding Options from the ... menu. Make sure that embedding is enabled for this data source and select '**Unauthenticated or Verified Signature**'
+
+4. Find the chart you want to embed, click the **...** menu and select **Embed Chart**
+
+5. Ensure the Unauthenticated tab is selected and turn on '**Enable unauthenticated access**'
+
+6. Select the **Javascript SDK** option
+
+7. Note the Chart ID and the Chart Base URL, as you will need them for running the demo.
+
+8. **Optional**
+   In the same menu, note the **User Specified Filters** option. If you wish to try out filtering on your own dataset, you will need to whitelist a field by which to filter on. For example, our sample AirBnB dataset filters on `address.country`.
+   Furthermore, the filter related code in `src/index.js` will need to be updated to conform to the filter query you wish to run, and the options provided in `index.html` will need to be updated too. To be clear,
+   - Update the query **field** in `src/index.js`
+   - Update the query **values** in `index.html`
+
+## Running this Sample
+
+---
+
+_The following steps presume the use of npm, though yarn works as well._
+
+1. Ensure you have Node installed. You can confirm with `node --version`. On some operating systems, Node available as the `nodejs` binary instead.
+
+2. Clone the Git repository or download the code to your computer.
+
+3. **Optional**
+   If you do not wish to use our sample data and have completed the above steps to prepare your own chart for embedding,
+   - Open the _index.js_ file (`src/index.js`)
+   - Replace the `baseUrl` string on `line 7` with the base URL you copied from the MongoDB Charts Embedded Chart menu.
+   - Replace the `chartID` string on `line 12` with the chartID you copied from the MongoDB Charts Embedded Chart menu.
```

Comment:
The hard-coded line numbers aren't very maintainable (I can almost guarantee that at some point the sample will be modified but the line numbers in the readme won't be updated). Can we modify the sample to have something like:

```
  baseUrl: "https://charts.mongodb.com/charts-tomh-yfpqi" // ~REPLACE~ with the Base URL from your Embed Chart dialog
...
  chartId: "92f7af7d-de64-4772-9bf4-b6fd2f59ac59", // ~REPLACE~ with the Chart ID from your Embed Chart dialog
```
And then update the content with something like:

> - Replace the `baseUrl` string on with the base URL you copied from the MongoDB Charts Embedded Chart menu (look for "~REPLACE~" in the comment)